api/CcsSso.Shared/Domain/Helpers/UtilityHelper.cs

```diff
@@ -24,15 +24,30 @@ public static bool IsEmailFormatValid(string emailaddress)
 
     public static bool IsEmailLengthValid(string email)
     {
-      if (email.Length > Constants.Constants.EmailMaxCharacters ||
-        email.Split('@')?.First()?.Length > Constants.Constants.EmailDomainMaxCharacters)
+     try
+      {
+        string[] userIdDomain_SubDomain = email.Split('.');
```

Comment:
please test for email addresses like firstname.lastname@domain.com (Note there is a "." in firstname.lastname part)

Reply:
tested email addresses like [firstname.lastname@domain.com](mailto:firstname.lastname@domain.com). i have resolved the issue. 

Committed and pushed in same PR.